rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -544,6 +544,11 @@ error_code cellGameContentPermit(vm::ptr<char[CELL_GAME_PATH_MAX]> contentInfoPa
 			cellGame.error("cellGameContentPermit(): failed to initialize directory '%s' (%s)", dir, fs::g_tls_error);
 		}
 	}
+	else if (perm->can_create)
+	{
+		// Update PARAM.SFO
+		psf::save_object(fs::file(vfs::get(dir + "/PARAM.SFO"), fs::rewrite - fs::create), perm->sfo);
```

Comment:
If you remove create flag, it will just crash in case file doesn't exist.

Reply:
It should exist here I think.

Reply:
`if (perm->can_create && perm->temp.empty() && !fs::is_dir(vfs::get(dir)))` should take care for this case.

Reply:
Modified to `perm->can_create && perm->temp.empty() && perm->exists` to handle better this case.